### ISSUE TITLE
About Encoding

### ISSUE THREAD

**Issue by batutahibnu17** [taspinar/twitterscraper#147]
_twitterscraper -u example -o example.txt_

```
INFO: Got 652 tweets from username example
Traceback (most recent call last):
  File "/usr/local/bin/twitterscraper", line 11, in <module>
    sys.exit(main())
  File "/usr/local/lib/python2.7/dist-packages/twitterscraper/main.py", line 105, in main
    with open(args.output, "w", encoding="utf-8") as output:
TypeError: 'encoding' is an invalid keyword argument for this function
```

Im install twitterscrapper from Pip (pip install twitterscrapper)

**Comment by vinaybommana** [comment]
```
Traceback (most recent call last):
  File "/usr/bin/twitterscraper", line 11, in <module>
    load_entry_point('twitterscraper==0.9.0', 'console_scripts', 'twitterscraper')()
  File "/usr/lib/python2.7/site-packages/twitterscraper/main.py", line 105, in main
    with open(args.output, "w", encoding="utf-8") as output:
TypeError: 'encoding' is an invalid keyword argument for this function

```

got the same error here

**Comment by vinaybommana** [comment]
I've just changed the following line

```
with open(args.output, "w", encoding="utf-8") as output:
```
to

```
with open(args.output, "wb") as output:
```

and added another line

```
output = output.read().decode("UTF-8")
```
that seems to work for me

overall the code looks like this in `main.py`

```
            if tweets:
                with open(args.output, "wb") as output:
                    if args.csv:
                        output = output.read().decode("UTF-8")
                        f = csv.writer(output)
                        f.writerow(["user", "fullname", "tweet-id", "timestamp", "url", "likes", "replies", "retweets", "text", "html"])

```

**Comment by LucieLoubere** [comment]
thank vinaybommana, 
your script work for me :)
Think to re-install the twitterscraper after change the main.py.

**Comment by vinaybommana** [comment]
Hi LucieLoubere,

Can you try this:

```
with open(args.output, "wb+") as output:
```

**Comment by 3ruce** [comment]
ok, that worked for me with a quick test... thank you...

**Comment by taspinar** [comment]
This issue should be fixed in version 0.9.3 which uses codecs.open() to open a file instead of open().

**Comment by 3ruce** [comment]
Running 0.9.3 I didn't see this updated code, or if it was updated, it didn't work...

I had to add these lines as mentioned above to the [23rd July commit](https://github.com/taspinar/twitterscraper/commit/5396f8e87421239e26af90885a4a24eb79819eae) and install via `sudo python setup.py install` rather than `sudo pip install twitterscraper`

```
with open(args.output, "wb+") as output:
        if args.csv:
            output = output.read().decode("UTF-8")
```
